server/app/services/cloud/aws/AwsApplicantStorage.java

```diff
@@ -181,13 +179,11 @@ class LocalStackClient implements Client {
     LocalStackClient(Config config, AwsStorageUtils awsStorageUtils) {
       this.config = checkNotNull(config);
       this.awsStorageUtils = checkNotNull(awsStorageUtils);
-      String localEndpoint = awsStorageUtils.localStackEndpoint(config);
-      try {
-        URI localUri = new URI(localEndpoint);
-        presigner = S3Presigner.builder().endpointOverride(localUri).region(region).build();
-      } catch (URISyntaxException e) {
-        throw new RuntimeException(e);
-      }
+      this.presigner =
```

Comment:
question (non-blocking): I feel like this is my own lack of Java knowledge, but why did we assign it to `presigner` before and now we're assigning it to `this.presigner`?

Reply:
sounds good to me! thanks for explaining.